bin/pygrb/pycbc_pygrb_efficiency

```diff
@@ -428,48 +417,48 @@ if do_injections:
     # NOTE: the loop on num_mc_injs would fill up the *_inj['dist_mc']'s at the
     # same time, so filling them up sequentially will vary the numbers a little
     # (this is an MC, order of operations matters!)
-    found_inj['dist_mc'] = ppu.mc_cal_wf_errs(num_mc_injs, found_inj['dist'],
+    found_inj_dist_mc = ppu.mc_cal_wf_errs(num_mc_injs, injs['found/distance'],
                                               cal_error, wav_err, max_dc_cal_error)
-    missed_inj['dist_mc'] = ppu.mc_cal_wf_errs(num_mc_injs, missed_inj['dist'],
+    missed_inj_dist_mc = ppu.mc_cal_wf_errs(num_mc_injs, injs['missed/distance'],
                                                cal_error, wav_err, max_dc_cal_error)
     long_inj['dist_mc'] = ppu.mc_cal_wf_errs(num_mc_injs, long_inj['dist'],
                                              cal_error, wav_err, max_dc_cal_error)
 
-    logging.info("MC injection set distributed with %d iterations.",\
+    logging.info("MC injection set distributed with %d iterations.",
                  num_mc_injs)
 
     # Check injections against on source
-    more_sig_than_onsource = np.ndarray(len(found_injs))
+    more_sig_than_onsource = np.ndarray(len(injs['found/distance']))
```

Comment:
You can actually remove the whole line, as `more_sig_than_onsource` will be redefined anyway on lines 433 or 435.